src/firefly/js/api/ApiHighlevelBuild.js

```diff
@@ -123,10 +127,12 @@ function buildTablePart(llApi) {
      */
 
     /**
-     * The general plotting function to plot a table.
+     *@function showTable
      * @param {string|HTMLDivElement} targetDiv to put the table in.
      * @param {Object} request         request object created from
      * @param {TblOptions} options     table options.
+     * @memeberof firefly
+     * @example fierfly.showTable
```

Comment:
This is nice.  It works great for the these API functions.  Too bad it does not recognize TblOptions @typedef which is declared just above it.  We'll have to figure out how to pull these typefefs in as well.